lib/editor.js

```diff
@@ -100,7 +100,9 @@ export default class Editor {
     this.emitter.emit('did-destroy')
     this.subscriptions.dispose()
     if (this.gutter) {
-      this.gutter.destroy()
+      try {
+        this.gutter.destroy()
+      } catch (_) { /* No Op */ }
```

Comment:
Maybe put in a comment that this is a workaround for what appears to be an Atom internal bug?